src/pretix/control/views/global_settings.py

```diff
@@ -60,7 +60,7 @@ def get_context_data(self, **kwargs):
         return ctx
 
     def get_success_url(self):
-        return reverse('control:global.update')
+        return reverse('control:global.update')    
```

Comment:
Revert please, purely whitespace change. Makes merging patches more complicated in the future.

Reply:
ok will revert those changes and update the branch

Reply:
but amazed that an extra space can also cause conflicts.

Reply:
patch does compare the lines as they are there and the lines as saved in the patch file.
If there is a difference, even wrt whitespace, patch/git has extra options to ignore whitespace, but that is dangerous, since Python is white space depending language

Reply:
ok,will always keep next time.